### ISSUE TITLE
Finalize Configuration File Values for 2021 Competition

### ISSUE THREAD

**Issue by mmartin4972** [umrover/mrover-workspace#517]
After final integration testing and system tuning, finalize all our configuration values. This issue should not be completed until just before 2021 competition. 

- [ ] Test and tune all configuration file values
- [ ] Upon completion, create a new issue for the 2021-2022 season.

Currently blocked on #474

**Comment by mmartin4972** [comment]
Make sure to test the max number of points in ECE. We will want to increase this value in the configuration file
